packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -899,6 +900,38 @@ const Conversation = WebexPlugin.extend({
 
     // triggers user-activity to reset logout timer
     this.webex.trigger('user-activity');
+    if (activity.target && !activity.target.defaultActivityEncryptionKeyUrl) {
+      const updateKeyActivity = {
+        // Needed to display in the stream instantly
+        actor: {
+          objectType: ConversationConstant.conversation.updateKeyActivity.PERSON,
+          id: activity.actor.id
+        },
+        // Needed for round trip
+        // clientTempId: API.TEMP_ID_PREFIX + uuid.v4(),
+        // Minimum properties needed by API
+        object: {
+          displayName: ConversationConstant.conversation.updateKeyActivity.UPDATE_KEY,
+          content: ConversationConstant.conversation.updateKeyActivity.UPDATE_KEY,
+          objectType: ConversationConstant.conversation.updateKeyActivity.CONVERSATION
+        },
+        target: {
+          id: activity.target.id, // accept convo object or ID
+          objectType: ConversationConstant.conversation.updateKeyActivity.CONVERSATION,
+          url: activity.target.url
+        },
+        verb: ConversationConstant.conversation.updateKeyActivity.UPDATE_KEY,
+        published: new Date().toISOString(),
+        clientPublished: new Date().toISOString(), // use to bundle correctly in the activitylist
+        _status: 'PENDING'
+      };
+
+      return Promise.resolve(this.webex.internal.encryption.kms.updateKmsKey(updateKeyActivity)).then((updateKmsResponse) => {
```

Comment:
To improve the readability of this section, please refactor based on line-length

usable example:

```js
return Promise.resolve(
  this.webex.internal.encryption.kms.updateKmsKey(updateKeyActivity)
  ).then((updateKmsResponse) => {
    // Destructure for improved readability.
    const {defaultActivityEncryptionKeyUrl} = updateKmsResponse.target;

    activity.target.defaultActivityEncryptionKeyUrl =
      defaultActivityEncryptionKeyUrl;

    return this.submit(activity);
  });
```

Reply:
okay, will update it.